docs/standard/serialization/snippets/system-text-json-how-to/csharp/WeatherForecastRequiredPropertyConverterForAttributeRegistration.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace SystemTextJsonSamples
+{
+    public class WeatherForecastRequiredPropertyConverterForAttributeRegistration : JsonConverter<WeatherForecastWithReqPptyConverterAttribute>
+    {
+        public override WeatherForecastWithReqPptyConverterAttribute Read(
+            ref Utf8JsonReader reader,
+            Type type,
+            JsonSerializerOptions options)
+        {
+            // OK to pass in options when recursively calling Deserialize.
+            WeatherForecastWithReqPptyConverterAttribute forecast = JsonSerializer.Deserialize<WeatherForecastWithoutReqPptyConverterAttribute>(ref reader);
+
+            // Check for required fields set by values in JSON
```

Comment:
```suggestion
            // Check for required fields set by values in JSON.
```

For consistency with other comments.